CRM/Activity/Import/Form/DataSource.php

```diff
@@ -20,7 +20,11 @@
  */
 class CRM_Activity_Import_Form_DataSource extends CRM_Import_Form_DataSource {
 
-  const PATH = 'civicrm/import/activity';
+  /**
+   * Variable to store redirect path.
+   * @var string
+   */
+  protected $_userContext = 'civicrm/import/activity';
```

Comment:
@mattwire I took a look at this & all the userContext stuff can be handled by this one-liner

https://github.com/civicrm/civicrm-core/pull/25600/files#diff-f21fc18c9e7abe639a59c87eb679d2983815b234c2282897eeb57086c26a7888R32

If, as I suspect, my alternate PR gets irrevocably bogged down by the handling for the Exception I put in there we can update this PR with that - in the meantime I'll put up a PR to remove `PATH` and `IMPORT_ENTITY` per this PR

Reply:
OK - this PR does that https://github.com/civicrm/civicrm-core/pull/25601